package.json

```diff
@@ -7,6 +7,9 @@
     "eslint": "node_modules/.bin/eslint generators/ test/",
     "cobertura": "istanbul report cobertura --root coverage"
   },
+  "engines": {
+    "node": "7.7.4"
```

Comment:
i actually don't think this is correct, yarn complains if you're not using the exact versions required in package.json, and we usually work with 6.10